### ISSUE TITLE
[BUG] Enabling watchdog in systemd makes maestral crash

### ISSUE THREAD

**Issue by peterhoeg** [SamSchott/maestral-dropbox#62]
**Describe the bug**

With version 0.4.1, everything works fine as long as I don't enable the watchdog. 

I added this, as an override file in ```/run/user/1000/system/user/maestral.service.d/override.conf```:

```
[Service]
WatchdogSec = 30s
```

And now maestral refuses to start:
```
Oct 22 21:59:54 dolores systemd[1126]: Starting Maestral - a Dropbox clone...
Oct 22 21:59:55 dolores systemd[1126]: Started Maestral - a Dropbox clone.
Oct 22 21:59:55 dolores maestral[5770]: Traceback (most recent call last):
Oct 22 21:59:55 dolores maestral[5770]:   File "/nix/store/vrllbj5ial3g70b2wd2bxmx01rxv32yw-maestral-0.4.1/lib/python3.7/site-packages/maestral/sync/daemon.py", line 119, in start_maestral_daemon
Oct 22 21:59:55 dolores maestral[5770]:     m = ExposedMaestral(run=run)
Oct 22 21:59:55 dolores maestral[5770]:   File "/nix/store/vrllbj5ial3g70b2wd2bxmx01rxv32yw-maestral-0.4.1/lib/python3.7/site-packages/maestral/sync/main.py", line 255, in __init__
Oct 22 21:59:55 dolores maestral[5770]:     self.update_thread.start()
Oct 22 21:59:55 dolores maestral[5770]:   File "/nix/store/8lhmyjarm73453f2mdz0xli9w8sy0wvh-python3-3.7.4/lib/python3.7/threading.py", line 848, in start
Oct 22 21:59:55 dolores maestral[5770]:     raise RuntimeError("threads can only be started once")
Oct 22 21:59:55 dolores maestral[5770]: RuntimeError: threads can only be started once
```
**To Reproduce**
Add watchdogsec to service.

**Expected behaviour**
For it to not blow up.

**System (please complete the following information):**
 - Maestral version: 0.4.1
 - OS:
 - system: `"x86_64-linux"`
 - host os: `Linux 5.3.7, NixOS, 20.03.git.c02c4775b40M (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.1`
 - channels(root): `""`
 - channels(peter): `"home-manager"`
 - nixpkgs: `/home/peter/src/active/nixpkgs_unstable`
 - Desktop environment: KDE Plasma 5.16
 - Qt version: 5.12.3

**Comment by SamSchott** [comment]
> For it to not blow up.

That is a fair expectation. I will have to look into that.

**Comment by SamSchott** [comment]
Ok, there were two really stupid bugs with the watchdog code:

1. A type in the name of thread which send WATCHDOG=1 notifications.
2. A conversion error from usec to sec when calculating the interval for notifications.

Both are fixed in the development branch.

**Comment by SamSchott** [comment]
fixed in v0.4.2
